infra/src/dsl/xroad.ts

```diff
@@ -172,9 +172,9 @@ export const RskProcuring = new XroadConf({
         '["clustercfg.general-redis-cluster-group.dnugi2.euw1.cache.amazonaws.com:6379"]',
     },
     XROAD_RSK_PROCURING_PATH: {
-      dev: 'IS-DEV/GOV/10006/Skatturinn/prokura-v1',
-      staging: 'IS-TEST/GOV/5402696029/Skatturinn/prokura-v1',
-      prod: 'IS/GOV/5402696029/Skatturinn/prokura-v1',
+      dev: 'IS-DEV/GOV/10006/Skatturinn/relationships-v1',
+      staging: 'IS-TEST/GOV/10006/Skatturinn/relationships-v1',
```

Comment:
Same here

Reply:
Rsk has not responded yet but I updated staging and prod to use RSK ssn